samples/suit/recovery/README.rst

```diff
@@ -30,41 +30,46 @@ It is optimized for memory usage and only contains the basic necessary functiona
     This firmware is only able to recover from a situation where the application or radio core are damaged it does not recover from Nordic Semiconductor-controlled firmware failures.
 
 Configuration
-**************
+*************
 
-As the recovery firmware is a companion image, it  must be compatible with the main application in terms of hardware configuration (especially the memory partitions).
+|config|
+
+As the recovery firmware is a companion image, it must be compatible with the main application in terms of hardware configuration (especially the memory partitions).
 To achieve this, the appropriate devicetree overlay files from the main application must be passed to the recovery application.
 
-To do this, add the :file:`recovery.overlay` and :file:`recovery_hci_ipc.ovelay` files in the main application's Sysbuild directory.
+To do this, add the :file:`recovery.overlay` and :file:`recovery_hci_ipc.ovelay` files in the main application's :ref:`configuration_system_overview_sysbuild` directory.
 The former file will be passed automatically to the recovery application image and the latter to the recovery radio image.
 These devicetree files must define the ``cpuapp_recovery_partition`` and ``cpurad_recovery_partition`` nodes respectively.`
 For an example, see the files in the ``samples/suit/smp_transfer`` sample.
 
 Building and running
 ********************
 
+.. |sample path| replace:: :file:`samples/suit/recovery`
+
+This sample can be found under |sample path| in the |NCS| folder structure.
+
+Setting build recovery Kconfig option
```

Comment:
I do not like this title, this section contains more information then only setting the option.

Can this title be changed to
"Building as part of the main application build"
?

Reply:
Ah, right, I'll fix.

Reply:
Other alternative: "Including recovery application image with sysbuild".
If there is a common syntax to express enabling additional images through sysbuild Kconfig option, it should work too.